packages/gatsby-cli/src/reporter/index.ts

```diff
@@ -231,21 +286,24 @@ const reporter: Reporter = {
 
     const span = tracer.startSpan(text, spanArgs)
 
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
+    const reporter = this
```

Comment:
I don't think this is used?

Reply:
oh, nvm it is. But I think we should revert to an object instead of a class.

Reply:
I made it a class to avoid putting loads of side effects in the top level. Keeping them within the class - even if it is instantiated on import - makes it more explicit what the side effects are. In a future updadate I'd hope we could export a singleton factory instead, and remove import side effects entirely